fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCMetadata.java

```diff
@@ -121,4 +121,7 @@ public Table getTable(String dbName, String tblName) {
         }
     }
 
+    private String isIgnorecase(String value) {
+        return (Boolean.valueOf(properties.getOrDefault("lower_case_table_names", "true")) ? value.toLowerCase() : value);
+    }
```

Comment:
may be we can just return the original db name and table name without toLowerCase? otherwise we can't get right name. How do you think about it?

Reply:
@zombee0  i think it is ok.  done